src/main/java/io/stargate/sgv2/jsonapi/service/operation/collections/CreateCollectionOperation.java

```diff
@@ -402,9 +440,15 @@ TableMetadata findTableAndValidateLimits(
     return null;
   }
 
-  public SimpleStatement getCreateTable(String keyspace, String table) {
+  public static SimpleStatement getCreateTable(
```

Comment:
Changing this to `static` since we should be using that from some of Integration tests (unfortunately we have copy-pasted versions). Also ensures all arguments are explicitly passed since call paths differ a bit wrt overrides, specifically "lexicalConfig".